standalone-metastore/metastore-rest-catalog/src/test/java/org/apache/iceberg/rest/TestHMSCatalogAuthNone.java

```diff
@@ -36,45 +39,41 @@
 import org.apache.iceberg.rest.requests.CreateTableRequest;
 import org.apache.iceberg.rest.requests.RenameTableRequest;
 import org.apache.iceberg.types.Types;
-import static org.apache.iceberg.types.Types.NestedField.required;
 import org.junit.After;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
 
-public class TestHMSCatalog extends HMSTestBase {
-  public TestHMSCatalog() {
-    super();
-  }
-  
+public class TestHMSCatalogAuthNone extends HMSTestBase {
   @Before
   @Override
   public void setUp() throws Exception {
-      super.setUp();
+    conf = MetastoreConf.newMetastoreConf();
+    MetastoreConf.setVar(conf, MetastoreConf.ConfVars.ICEBERG_CATALOG_SERVLET_AUTH, "none");
+    super.setUp();
   }
-  
+
   @After
   @Override
   public void tearDown() throws Exception {
-      super.tearDown();
+    super.tearDown();
   }
 
   @Test
   public void testCreateNamespaceHttp() throws Exception {
     String ns = "nstesthttp";
     // list namespaces
     URL url = new URL("http://hive@localhost:" + catalogPort + "/"+catalogPath+"/v1/namespaces");
-    String jwt = generateJWT();
     // check namespaces list (ie 0)
-    Object response = clientCall(jwt, url, "GET", null);
+    Object response = clientCall(url, "GET", Collections.emptyMap(), null);
```

Comment:
is that the only difference? 
if yes, can't we move the tests into base class and override just the client function call?

Reply:
It's okay if it's a blocker, but I'm removing the current test case in this pull request. I'll do it if we want to merge HIVE-28998 first, and the change is a must.
https://github.com/apache/hive/pull/5887

Reply:
Let's merge https://github.com/apache/hive/pull/5887 first.